Thinktecture.Relay.Server.Test/Diagnostics/TraceManagerTest.cs

```diff
@@ -26,12 +24,17 @@ private class OnPremiseTargetResponse : IOnPremiseTargetResponse
 		{
```

Comment:
This class seems to have no references anymore.

Reply:
Changed to implement the used object